deployment/helm/templates/nginx-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.settings.name }}
+  labels:
+    app.kubernetes.io/name: {{ .Values.settings.name }}
+    app.kubernetes.io/part-of: {{ .Values.settings.name }}
+    helm.sh/chart: {{ include "chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  type: NodePort
```

Comment:
this may conflict with other http servers on those nodes - the default is ClusterIP and intended to sit behind an ingress. Maybe make this a variable?